pkgs/applications/networking/p2p/gnunet/gtk.nix

```
@@ -31,10 +33,14 @@ stdenv.mkDerivation rec {
     gtk3
     libextractor
     libgcrypt
+    libqrencode
```

```suggestion
    qrencode
```

Don't use aliases internally.